File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c#/Angel-Delg.cs
using System;

namespace RETO_0 {
   class Root{
      static void Main(){
         FizzBuzz();
      }

      static void FizzBuzz(){
         for(int i = 0; i < 100; i++){
            if((i + 1) % 3 == 0 && (i + 1) % 5 == 0){
               Console.WriteLine("FizzBuzz");
               continue;
            }
            else if((i + 1) % 5 == 0){
               Console.WriteLine("Buzz");
               continue;
            }
            else if((i + 1) % 3 == 0){
               Console.WriteLine("Fizz");
               continue;
            }
            else{
               Console.WriteLine(i + 1);
            }
         }
      }
   }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/typescript/Angel-Delg.ts
type Output = "Fizz" | "Buzz" | "FizzBuzz" | number; 

const FizzBuzz = ():void => {
   for(let i = 1; i <= 100; i++){
      const output: Output = (i % 3 === 0 && i % 5 === 0) ? "FizzBuzz"
      : (i % 3 === 0) ? "Fizz" : (i % 5 === 0) ? "Buzz" : i
      console.log(output)
   }
} 